README.md

```diff
@@ -3,18 +3,47 @@ A list of resources for data science work on climate change
 
 ## Data Sets
 
+### Global
+* [ICE Embodied Carbon Database](https://circularecology.com/embodied-carbon-footprint-database.html)
+* [Carbon in food downloadable csv](https://ourworldindata.org/food-choice-vs-eating-local)
+* [EPW Weather files (for Ladybug on Grasshopper, but adaptable)](https://www.ladybug.tools/epwmap/)
+* [Renewable Ninja - front end for NASA dataset](https://www.renewables.ninja/)
+* [One of many NASA datasets. This one covers precipitation](https://disc.gsfc.nasa.gov/datasets/GPM_3IMERGHH_06/summary?keywords=IMERG)
+
+* [ Keen cyclist route heatmap](https://www.strava.com/heatmap)
```

Comment:
this isn't really a data set or a tool related *specifically* to climate change, or it's remediation, can we remove it please?